extension/src/config/config.js

```diff
@@ -27,6 +54,15 @@ function getCtpConfig(ctpProjectKey) {
       ctpConfig.apiUrl || 'https://api.europe-west1.gcp.commercetools.com',
     authUrl:
       ctpConfig.authUrl || 'https://auth.europe-west1.gcp.commercetools.com',
+    authScheme: ctpConfig.authentication
```

Comment:
Use optional chaining instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining

Sth like `ctpConfig.authentication?.scheme`

Reply:
Optionl chaining is only compatible with node v.14 or above. Shall we change to it?

Reply:
Yes, we require explicitly node v14 or higher :) https://github.com/commercetools/commercetools-adyen-integration/blob/master/extension/package.json#L40